contour/reference/index.hbs.md

```diff
@@ -0,0 +1,14 @@
+# Contour Reference
+
+This section contains reference documentation for using Contour.
+
+## Open Source Documentation
+
+The [Contour Open Source Documentation](https://projectcontour.io/docs/1.25/) contains more detailed, Contour-specific reference information. This is best for more advanced usages of Contour.
+
+**WARNING:** There may be some features or configurations that aren't directly accessible via the Contour Package API in TAP.
+
+Some sections of interest:
+* [API Reference](https://projectcontour.io/docs/1.25/config/api/)
+* [Configuration Reference](https://projectcontour.io/docs/1.25/configuration/)
+* [Compatability Matrix](https://projectcontour.io/resources/compatibility-matrix/)
```

Comment:
suggestion: Can we provide the links without the version number so we don't need to update these links every release? We can preface saying to match the version to the version used by TAP or something

Reply:
how are you imagining it to look?

I don't know if I can get version-agnostic links

Reply:
Yeah, only up to `/docs` can be versionless. Nevermind then :)